PKGBUILD

```diff
@@ -32,8 +32,8 @@ source=(
 )
 
 sha256sums=('0abf1e8da4676363492a09e6d5810c50f0b9d053cab81a35b4005faafa1eebbd'
-            'abefbc697ebab05556f955a66b634bcd30420ca7632901f648f20c23b181a041'
```

Comment:
I'm sorry about this change, apparently `makepkg` adds  `SKIP` to signature/checksum files.